### ISSUE TITLE
Recall on ICDAR2015 dataset

### ISSUE THREAD

**Issue by lhao0301** [WenmuZhou/PytorchOCR#110]
training setting:
icdar2015
resnet-50(pre-trained)

result:
the best model on icdar2015 validation set achieves recall=75.9%, precision=~88.xx%, hmean=~81.xx%

question:
recall = 75.9% vs 79.9%(reported in this repo) vs 82.7%(reported in official paper)

recall is a little lower than both in this repo and official paper. Is this normal?

**Comment by lhao0301** [comment]
R=82.7% is achieved after pre-trainning on SynthText dataset as stated in original paper(Implementation details section).
